ofborg/src/bin/github-comment-filter.rs

```diff
@@ -1,91 +1,78 @@
+        exchange_type: easyamqp::ExchangeType::Topic,
+        passive: false,
+        durable: true,
+        auto_delete: false,
+        no_wait: false,
+        internal: false,
+    })?;
 
-    channel
-        .declare_exchange(easyamqp::ExchangeConfig {
-            exchange: "build-jobs".to_owned(),
-            exchange_type: easyamqp::ExchangeType::Fanout,
-            passive: false,
-            durable: true,
-            auto_delete: false,
-            no_wait: false,
-            internal: false,
-        })
-        .unwrap();
+    chan.declare_exchange(easyamqp::ExchangeConfig {
+        exchange: "build-jobs".to_owned(),
+        exchange_type: easyamqp::ExchangeType::Fanout,
+        passive: false,
+        durable: true,
+        auto_delete: false,
+        no_wait: false,
+        internal: false,
+    })?;
 
-    channel
-        .declare_queue(easyamqp::QueueConfig {
-            queue: "build-inputs".to_owned(),
-            passive: false,
-            durable: true,
-            exclusive: false,
-            auto_delete: false,
-            no_wait: false,
-        })
-        .unwrap();
+    let queue_name = "build-inputs";
+    chan.declare_queue(easyamqp::QueueConfig {
+        queue: queue_name.to_owned(),
+        passive: false,
+        durable: true,
+        exclusive: false,
+        auto_delete: false,
+        no_wait: false,
+    })?;
 
-    channel
-        .bind_queue(easyamqp::BindQueueConfig {
+    chan.bind_queue(easyamqp::BindQueueConfig {
+        queue: "build-inputs".to_owned(),
+        exchange: "github-events".to_owned(),
+        routing_key: Some("issue_comment.*".to_owned()),
+        no_wait: false,
+    })?;
+
+    let handle = chan.consume(
+        tasks::githubcommentfilter::GitHubCommentWorker::new(cfg.acl(), cfg.github()),
+        easyamqp::ConsumeConfig {
             queue: "build-inputs".to_owned(),
-            exchange: "github-events".to_owned(),
-            routing_key: Some("issue_comment.*".to_owned()),
+            consumer_tag: format!("{}-github-comment-filter", cfg.whoami()),
+            no_local: false,
+            no_ack: false,
             no_wait: false,
-        })
-        .unwrap();
-
-    channel.basic_prefetch(1).unwrap();
-    let mut channel = channel
-        .consume(
-            worker::new(tasks::githubcommentfilter::GitHubCommentWorker::new(
-                cfg.acl(),
-                cfg.github(),
-            )),
-            easyamqp::ConsumeConfig {
-                queue: "build-inputs".to_owned(),
-                consumer_tag: format!("{}-github-comment-filter", cfg.whoami()),
-                no_local: false,
-                no_ack: false,
-                no_wait: false,
-                exclusive: false,
-            },
-        )
-        .unwrap();
-
-    channel.start_consuming();
+            exclusive: false,
+        },
+    )?;
 
-    info!("Finished consuming?");
+    info!("Fetching jobss from {}", &queue_name);
```

Comment:
```suggestion
    info!("Fetching jobs from {}", &queue_name);
```

Unless borg is a snake now.